_data/sites.json

```
@@ -2939,8 +2939,7 @@
         "name": "Chegg",
         "url": "https://www.chegg.com/contactus",
         "difficulty": "hard",
-        "notes": "Call customer service, email them with the email listed here, contact them via chat and ask them to delete your account, or request assistance via their Twitter account, <a href=\"https://twitter.com/chegghelp\">@CheggHelp</a>. There is also no option to delete your payment methods, you need to contact customer support as well.",
-        "notes_tr": "Müşteri hizmetlerini arayın ya da kendileriyle mesaj olarak iletişime geçin ve hesabınızı silmek istediğinizi belirtin, ya da kendilerinin Twitter hesabıyla yardım isteyin. Ayrıca ödeme yöntemlerinizi silmek için bir yöntem yok, bunun için de müşteri hizmetleriyle iletişime geçmeniz gerekir.",
+        "notes": "<ul><li><strong>To request data deletion:</strong> Use the <a href=\"https://privacyportal-cdn.onetrust.com/dsarwebform/a21a74f4-2f93-44a2-a887-302a9213c997/06b040b7-49eb-464a-b19a-ea35c85c79fc.html\">Data Deletion Form</a></li><br><li><strong>To delete payment methods:</strong> Contact support using the SMS/WhatsApp number (408)-351-0942. You will need the last four digits of the payment method.</li><br><li><strong>To delete/deactivate account:</strong> Contact support with the email listed here, call, message with SMS/WhatsApp, or request assistance via their twitter <a href=\"https://twitter.com/chegghelp\">@CheggHelp</a>.</li></ul>",
```

I think these notes are too long and the account deletion only comes last, unless you need to delete data and payment methods to then delete the account. I'm also not too fond of these many html elements inside the notes (even though I know they'll make reading easier on the actual website.

---

I agree that I don't like the length. Deleting payment methods and data are not entirely necessary to delete the account but they ensure all data is removed. Would this work better?

```suggestion
        "notes": "Contact support with the email listed here, call or message (408)-351-0942, or request assistance via their twitter <a href=\"https://twitter.com/chegghelp\">@CheggHelp</a> for account closing and payment method removal. For data deletion click <a href=\"https://privacyportal-cdn.onetrust.com/dsarwebform/a21a74f4-2f93-44a2-a887-302a9213c997/06b040b7-49eb-464a-b19a-ea35c85c79fc.html\">here.",
```

---

I think it does, thanks.